--- rtai-3.9.1.orig/m4/acinclude.m4
+++ rtai-3.9.1/m4/acinclude.m4
@@ -223,10 +223,10 @@ AC_CACHE_VAL(ac_cv_comedi,[
 
     # First check to see if --with-comedi was specified.
     if test x"${with_comedi}" != x ; then
-	if test -f "${with_comedi}/include/linux/comedilib.h" ; then
+	if test -f "${with_comedi}/include/comedilib.h" ; then
 	    ac_cv_comedi=`(cd ${with_comedi}; pwd)`
 	else
-	    AC_MSG_ERROR([${with_comedi}/include directory doesn't contain linux/comedilib.h])
+	    AC_MSG_ERROR([${with_comedi}/include directory doesn't contain comedilib.h])
 	fi
     fi
 
@@ -239,7 +239,7 @@ AC_CACHE_VAL(ac_cv_comedi,[
 		`ls -dr ../../comedi* 2>/dev/null` \
 		../../../comedi \
 		`ls -dr ../../../comedi* 2>/dev/null` ; do
-	    if test -f "$i/include/linux/comedilib.h" ; then
+	    if test -f "$i/include/comedilib.h" ; then
 		ac_cv_comedi=`(cd $i; pwd)`
 		break
 	    fi
@@ -250,7 +250,7 @@ AC_CACHE_VAL(ac_cv_comedi,[
     if test x"${ac_cv_comedi}" = x ; then
 	for i in ${prefix}/comedi* /usr/local/comedi* /usr/pkg/comedi* /usr \
 		`ls -dr /usr/lib/comedi* 2>/dev/null` ; do
-	    if test -f "$i/include/linux/comedilib.h" ; then
+	    if test -f "$i/include/comedilib.h" ; then
 		ac_cv_comedi=`(cd $i; pwd)`
 		break
 	    fi
@@ -262,7 +262,7 @@ AC_CACHE_VAL(ac_cv_comedi,[
 	for i in \
 		${srcdir}/../comedi \
 		`ls -dr ${srcdir}/../comedi* 2>/dev/null` ; do
-	    if test -f "$i/include/linux/comedilib.h" ; then
+	    if test -f "$i/include/comedilib.h" ; then
 	    ac_cv_comedi=`(cd $i; pwd)`
 	    break
 	fi
@@ -272,7 +272,7 @@ AC_CACHE_VAL(ac_cv_comedi,[
 
 if test x"${ac_cv_comedi}" = x ; then
     COMEDI_DIR="# no COMEDI installation found"
-    AC_MSG_ERROR(Can't find COMEDI installation ( missing linux/comedilib.h ) )
+    AC_MSG_ERROR(Can't find COMEDI installation ( missing comedilib.h ) )
 else
     COMEDI_DIR=${ac_cv_comedi}
     AC_MSG_RESULT(found $COMEDI_DIR)
